### PR TITLE
Refactor: Incorporate ButterKnife

### PR DESCRIPTION
**Pull request by samuelrey**
Bind member views using annotations to reduce boilerplate.
Separate function definition from on click binding.

### CODE REVIEW

**Comment by TheHollidayInn** [comment]
Awesome!! I’ll review in the next few days :)

**Comment by TheHollidayInn** [comment]
Alright, looks good! The PR was based on master though. Can you merge in upstream develop and fix the merge conflicts?

**Comment by samuelrey** [comment]
I'll figure out these merge problems!

File: app/src/main/java/com/koalatea/thehollidayinn/softwareengineeringdaily/podcast/PodcastAdapter.java [hunk 1]
```diff
@@ -17,13 +17,53 @@
...
+import butterknife.ButterKnife;
+
+/**
  * Created by krh12 on 5/22/2017.
  */
 
-class PodcastAdapter extends RecyclerView.Adapter<PodcastAdapter.ViewHolder> {
-  private List<Post> posts = new ArrayList<>();
-  private PodListFragment context;
+public class PodcastAdapter extends RecyclerView.Adapter<PodcastAdapter.ViewHolder> {
```
Comment (TheHollidayInn) [inline app/src/main/java/com/koalatea/thehollidayinn/softwareengineeringdaily/podcast/PodcastAdapter.java]: Looks like this file broke during merge. You might want to copy and paste from develop then reintroduce butterknife.

**Comment by TheHollidayInn** [comment]
Did you mean to close? @samuelrey

**Comment by samuelrey** [comment]
No, must have clicked it by accident.
I'm double checking my changes, I'll have them in soon!

**Comment by TheHollidayInn** [comment]
Sweet!

**Comment by samuelrey** [comment]
Where can I find the google-services.json file?

**Comment by TheHollidayInn** [comment]
For testing purposes, you now need to create an app on Firebase and then use the services file from that for your local build.

**Pull request merged**